cdap-docs/admin-manual/build.sh

```diff
@@ -80,9 +82,16 @@ function download_includes() {
   types="configuration hdfs-permissions"
   for type in ${types}; do
     rewrite_references_in_place_sed "${target_includes_dir}/mapr-${type}.rst" " su hdfs" " su mapr"
-    rewrite_references_in_place_sed "${target_includes_dir}/mapr-${type}.rst" "FQDN1:2181,FQDN2:2181" "FQDN1:5181,FQDN2:5181"
   done
   
+  local source_pattern="(FQDN1:2181,FQDN2:2181)"
+  local target_pattern=$(cat <<EOF
+(FQDN1:5181,FQDN2:5181);
```

Comment:
on the "page of interest", it is still showing ``<value>FQDN1:2181,FQDN2:2181/${root.namespace}</value>``, i think because of the parenthesis in the pattern?